tools/depends/xbmc-addons.include

```diff
@@ -93,6 +96,10 @@ $(TOOLCHAIN_FILE): $(abs_top_srcdir)/target/Toolchain_binaddons.cmake
 	sed "s|@CMAKE_FIND_ROOT_PATH@|$(ADDON_DEPS_DIR)|g" $(abs_top_srcdir)/target/Toolchain_binaddons.cmake > $@
 
 linux-system-libs:
+	mkdir -p $(ADDON_DEPS_DIR)/lib
+	[ -f $(ADDON_DEPS_DIR)/lib/libm.so ] || ln -sf /usr/lib/$(HOST)/libm.so $(ADDON_DEPS_DIR)/lib/
+
+linux-gl-libs:
```

Comment:
maybe call it linux-x11-gl-libs to make it explicit?
and linux-gbm-gles-libs respectively?

Reply:
We should really be building the gles lib for all linux gles platforms :smiley: It's not really about gbm

Reply:
then you should pull out the x11 stuff in it's own target

Reply:
sounds reasonable